kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/FitbitSourceConnector.java

```diff
@@ -98,7 +98,7 @@ public ConfigDef config() {
     try {
 
       List<Map<String, String>> userTasks = fitbitConfig.getUserRepository().stream()
-          .map(User::getId)
+          .map(User::getVersionedId)
```

Comment:
how do we handle existing users who do not have versionId? do we generate one by default for all if not found?

Reply:
The implementation is:
- has version? ID#Version
- otherwise: ID.
This makes it backwards compatible for unversioned users.